package.json

```diff
@@ -81,7 +81,9 @@
     "acceptance": "testcafe safari,chrome tests/acceptance/acceptancesuite.js",
     "size": "size-limit",
     "fix": "semistandard --fix",
-    "extract-translations": "gulp extractTranslations"
+    "extract-translations": "gulp extractTranslations",
```

Comment:
are these all the changes we need to publish to npm? I thought we might need to update `main` in the package.json as well, and maybe rebuild the sdk before publish?

Reply:
yeee, coming to you hot in another `dev` branch
for your latter thought, was thinking of a spruce-tools script, wdyt?

Reply:
I think the scripts in spruce-tools are for things we use manually, would we need to do anything besides an npm run build?

Reply:
Supposedly, we want this item to take care of https://yexttest.atlassian.net/browse/CPB-247. We don't want Consulting on dist tags any more. I'm not sure I follow the comment of CPB-247. Rose wants the package to do an `npm run build` after it's installed. That doesn't make a whole lot of sense to me. It seems like we should just publish what's in the `dist` directory to npm. We should follow up with her on that.

Reply:
`dist` is being added

Reply:
(in another PR)

Reply:
and there will be a new manual step to increment the `package.json` version and commit before our tagging process